extensions/amp-story/1.0/amp-story-interactive-poll.js

```diff
@@ -202,6 +205,21 @@ export class AmpStoryInteractivePoll extends AmpStoryInteractive {
         '.i-amphtml-story-interactive-option-percentage-text'
       ).textContent = `${percentage}%`;
       currOption.setAttribute('style', `flex-grow: ${percentage} !important`);
+
+      if (percentage < 20) {
+        const textContainer = currOption.querySelector(
+          '.i-amphtml-story-interactive-option-text-container'
+        );
+        textContainer.setAttribute(
```

Comment:
Missed this during hte previous review sorry, please use the setStyles() helper so you don't override already present styles

Reply:
Just commented this to the PR that we will actually use 😂